### DISCUSSION TITLE
Local Config Files using Synology Docker

### DISCUSSION THREAD

**Discussion by Bodo1706** [john30/ebusd#590] [Q&A] [answered]
Hi EBusser,

sorry for my stupid auestion, but I try now since days to get it fixed by my self without success.

I run a EBusD Container (V22.2) using Docker on my Synology DS220+. As the automatic configuration does not work perfectly with my Vaillant VRS620 I tried to load the configuration locally. But I have still a problem with the associated configuration. I get always the information "error reading config files from" (in the folder /usr/local/bin/ebusconf/EBusD/vaillant/ the csv files are stored e.g. "ec.solsy.sc.csv" :

2022-04-12 13:53:56 | stdout | 2022-04-12 15:53:56.937 [update notice] received unknown MS cmd: 1008b51009000000ffffff01ff00 / 0101
-- | -- | --
2022-04-12 13:53:56 | stdout | 2022-04-12 15:53:56.937 [bus notice] new master 03, master count 3
2022-04-12 13:53:56 | stdout | 2022-04-12 15:53:56.892 [bus notice] new master 10, master count 2
2022-04-12 13:53:52 | stdout | 2022-04-12 15:53:52.984 [bus notice] signal acquired
2022-04-12 13:53:52 | stdout | 2022-04-12 15:53:52.984 [bus notice] bus started with own address 31/36
2022-04-12 13:53:52 | stdout | 2022-04-12 15:53:52.957 [main error] error reading config files from /usr/local/bin/ebusconf/EBusD/vaillant/: ERR: element not found, last error:
2022-04-12 13:53:52 | stdout | 2022-04-12 15:53:52.957 [main notice] ebusd 22.2.p20220227 started on device tcp:192.168.178.26:5000

Here a picture of the EBusD configuration:

![grafik](https://user-images.githubusercontent.com/64847111/162979793-f8fc7578-8ab1-40f0-9d8b-9031fd0f7bee.png)

What can I do better?

Thanks for your help.

Bodo

**Comment by Koky05** [comment, marked as answer]
My configuration for DS720+, maybe it will help you:
`#eBus Deamon
  ebusd:
    container_name: ebusd
    image: john30/ebusd:latest
    depends_on:
      - mqtt
    #ports:
    #  - "8888:8888/tcp"
    command: [
            "-f",
            "--scanconfig=full",
            "--configpath=/etc/ebusd/ebusd-configuration/ebusd-2.1.x/en",
            "--pollinterval=15",
            "--port=8888",
            "--httpport=8082",
            "-d",
            "enh:192.168.1.12:9999",
            "--latency=150",
            "--mqtthost=192.168.1.2",
            "--mqttport=1883",
            "--mqttint=/etc/ebusd/mqtt-hassio.cfg",
            "--mqttjson",
            "--mqttverbose",
            "--mqttuser=hass",
            "--mqttpass=xxxxxxxx",
            "--mqttretain",
            "--mqttchanges",
            "--accesslevel=*",
            "--mqtttopic=ebusd"
      ]
    network_mode: host
    environment:
      TZ: 'Europe/Bratislava'
      EBUSD_ARCH: 'amd64'
    volumes:
      - "/volume1/docker/ebusd/data:/data"
      - "/volume1/docker/ebusd:/ebusd"
      - "/volume1/docker/ebusd/config:/etc/ebusd"
      - "/volume1/docker/ebusd/log:/var/log"
    labels:
      - "com.centurylinklabs.watchtower.enable=true"
    restart: unless-stopped`

**Reply by Bodo1706** [reply]
Hi Koky,
thx a lot. The connection between the path of the stored config to the mount path /etc/ebusd was the key. THX a lot.
Solved with the following entries (for people with the same problem - if any):

![grafik](https://user-images.githubusercontent.com/64847111/163269535-637ee68b-d99e-471d-b6f9-56333612cdff.png)

![grafik](https://user-images.githubusercontent.com/64847111/163270172-abe286cd-4ca1-4bf6-a596-70c4dc592237.png)


BR
Bodo
